search-books/src/main/java/nix/education/service/search/SolrSearchBookService.java

```diff
@@ -0,0 +1,83 @@
+package nix.education.service.search;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import nix.education.entity.Book;
+import nix.education.repository.BookRepository;
+import nix.education.repository.TagRepository;
+import nix.education.request.BookRequest;
+import nix.education.service.SavingBookService;
+import nix.education.service.query.SolrQueryBuilderService;
+import nix.education.service.response.SolrBookResponseService;
+import nix.education.service.suggestions.SolrBookAuthorSuggestionService;
+import nix.education.util.SymbolConstants;
+import org.apache.solr.client.solrj.SolrClient;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class SolrSearchBookService implements SearchService {
```

Comment:
I think we can just name this class SolrSearchService since the intention of service is clear from interface context

Reply:
Ok, agree

Reply:
Renamed.